src/main/java/com/aws/greengrass/componentmanager/plugins/docker/DockerImageDownloader.java

```diff
@@ -161,8 +162,11 @@ private File performDownloadSteps(Image image) throws PackageDownloadException,
                         // credentials for them on case by case basis
                         if (image.getRegistry().isEcrRegistry()) {
                             // Get auth token for ECR, which represents ECR registry credentials
+
+                            String actualRegion = getRegionFromArtifactUri(image.getArtifactUri().toString());
```

Comment:
What is `actualRegion`? it's clear in the context of this CR but won't be for someone who reads this code later. Let's find a better name for this that conveys this is the region where the image is hosted